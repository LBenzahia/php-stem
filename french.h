
/* This file was generated automatically by the Snowball to ANSI C compiler */

extern struct SN_env * french_create_env(void);
extern void french_close_env(struct SN_env * z);

extern int french_stem(struct SN_env * z);

