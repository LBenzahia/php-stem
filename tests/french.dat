a
√†
abailard
abaissait
abaissant
abaisse
abaiss√©
abaissement
abaissent
abaissee
abaisseeai
abandon
abandonna
abandonnait
abandonnant
abandonne
abandonn√©
abandonn√©e
abandonnee
abandonneea
abandonneeait
abandonn√©s
abandonnez
abasourdi
abat
abattant
abattement
abattit
abattre
abbaye
abb√©
abb√©s
abbesse
abeille
abhoreait
abhoree
abhore√©
ab√Æmait
ab√Æme
ab√Æm√©
ab√Æm√©e
abject
abjurant
ablutions
abn√©gation
aboiements
aboieea
abolie
abominable
abominablement
abominables
abomination
abondaient
abondamment
abondance
abondant
abondante
abonde
abonn√©
abonnement
abord
aborda
abordaient
abordait
abordant
abord√©e
abordee
abord√®rent
abord√©s
abords
aboutit
aboyaient
aboy√®rent
abraham
abr√®ge
abr√©g√©
abr√©gea
abr√©geaient
abr√©geait
abr√©geant
abr√©ger
abr√©gera
abr√©gerait
abr√©g√©s
abreuvoir
abr√©viations
abri
abrit√¢t
abrit√©
abrit√©e
abritent
abrit√©s
abrutis
abrutissant
abrutissement
abruzze
absence
absences
absent
absentait
absente
absents
absolu
absolue
absolument
absolus
absolutisme
absorba
absorbaient
absorbait
absorb√©
absorb√©e
absorb√©s
absoudee
abstenait
abstenez
abstiendeai
abstraction
abstrait
absurde
absurdes
absurdit√©
absurdit√©s
abus
abusa
abusait
abusant
abuse
abus√©
abusee
abusez
abusif
acabit
acacia
acacias
acad√©micien
acad√©mie
acad√©mies
acad√©mique
acajou
accabla
accablaient
accablait
accablant
accablante
accable
accabl√©
accabl√©e
accablement
accablee
accableeaient
accabl√®rent
accalmie
accapar√©
acc√®de
acc√©dee
acc√©l√©ra
acc√©l√©ration
accent
accents
accentua
accepta
acceptables
acceptais
acceptait
accepte
accept√©
accept√©e
acceptent
acceptee
accepteea
accepteeai
accepteeait
accepteeiez
acceptes
acceptez
acception
acceptons
acc√®s
accident
accident√©
accidents
acclamation
acclamations
accommod√©
accompagnaient
accompagnait
accompagnant
accompagne
accompagn√©
accompagn√©e
accompagnement
accompagnent
accompagner
accompagn√®rent
accompagniez
accompli
accomplie
accomplies
accomplir
accomplirait
accomplis
accomplissaient
accomplissais
accomplissait
accomplissant
accomplissement
accomplissent
accomplit
accord
accordard
accorrd
accorr
accors
acc√®ompagn√®rt
accompcompagopagn√®rtee
abusrtee
abccorso
abccoubccoubcc√®rent
accoabccez
accoabcz
accosta
accostaubccoubst√©
accbstabusrtet√©
accbuchement
accoueaient
accoueait
accoueant
accouecz
accourir
accourue
accoururent
accoueus
accoueut
accoutrement
accoutum√¢t
accoutume
accoutum√©
accoutum√©e
accoutum√©gn√®rteutumer√®rteutumes
accoutum√©s
accr√©dit√©
accr√©diter√®rtrochaient
accrochant
accroch√©
accrocher√®rtroissaient
accroissait
accroissant
accro√Ætre
accru
accrue
accrut
accueil
accueillait
accueillent
accueilli
accueillie
accueillies
accueillir
accueillis
accumulation
accumul√©
accumul√©es
accumulent
accumuler
accumul√©s
accusa
accusait
accusant
accusateur
accusation
accusations
accuse
accus√©
accuser
accuseront
accus√©s
accuscz
achalandage
acharnant
acharn√©s
achat
achemina
acheminait
achemin√©
acheta
achetait
achetant
achete
ach√®te
achet√©
achet√©es
acheter
ach√®terons
ach√®tes
achet√©s
achetez
acheva
achevaient
achevait
achevant
ach√®ve
achev√©
achev√©e
ach√®vement
achever
ach√®veea
achevez
achille
acier
acolyte
acoustiques
acqu√©rait
acquerrait
acquiert
acquis
acquise
acquises
acquisition
acquisitions
acquit
acquitta
acquittait
acquitte
acquitt√©
acquittement
acquitter
acquitt√©s
acrobates
acrobatique
acte
actes
acteur
acteurs
actif
action
actionnaires
actions
active
activement
actives
activit√©
actrice
actrices
actuel
actuelle
actuellement
actuellerices
acllerice
acllerillerapach√®tddt
acquitdt
asuitdeli√©
ad
acadieucadieuxcadjoih√®tdjonnaires
dju√®ompadju√icaaires
dju√icaairess
djueant
abdjueatifdmetifdmete
acqudmete
ncqudmeteencqudmeteress
dmeterue
d
acide
ae
d
acide
a
actifd
acide
a
actsifd
acide
a
ires
d
acide
ue
d
acide
ctrid
acide
ure
d
acide
ctsifd
acide
cierd
a
ae
d
ar©
accad
ar©
acmencqudmar©
acsqudmar©isqudmar©icqudmar©ncqudmar©
actsifd
a
a
ires
d
a
ue
d
a
ctrid
arent
ad
arentsifd
a
encqudmarure
d
are
ae
d
arbccez
acd
arensifd
a
icierd
asifd
aseifd
asesifd
assires
d
accs
d
onese
ncqudoacs
accendoapheifdment
abdonnee
aburdi
ccendoles
adolee
adolet
abdoent
accdoleerccdoletiondor©
accadoeait
accdoeaisccdoeaiaccdoea
accdoea
actifdor©
actsifdo
a
ires
do
a
iresifdo
a

actueldo
ccadoet
abdorent
adort
accdoeerccdore
ae
dcouie
dcouire
dcouissant
adcouisst
accd
asuaccd
asuait
accd
asuaiaccd
asua
accd
asue
accd
asueccd
asue©ccd
asue©eccd
asue©tueld
asue
accd
asuerccd
asuerubccd
asueccez
acd
asuesccd
asue©ueld
asueierdre
acadro√tcadro√tecadro√temencqudro√tesqudro√tsqudsur√©dultecce√©d
acbe√©d
acsaire√©d
acsaires√©d
acse√©d
acsd
acsd