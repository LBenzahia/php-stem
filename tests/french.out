a
Ã 
abailard
abaiss
abaiss
abaiss
abaiss
abaissement
abaissent
abaiss
abaiss
abandon
abandon
abandon
abandon
abandon
abandon
abandon
abandon
abandon
abandon
abandon
abandon
abasourd
abat
abatt
abatt
abatt
abattr
abbay
abbÃ©abbÃ©abbÃ©Ã©abbÃ©abbÃn©abbÃn©ala